packages/c/srv/src/srv.c

```diff
@@ -350,10 +350,13 @@ int socket_to_socket(const srv_params_t *params, const char *auth_string, chunke
   return 0;
 }
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-parameter"
 int server_to_socket(const srv_params_t *params, const char *auth_string, chunked_transformer_t *encrypter,
                      chunked_transformer_t *decrypter) {
   return 0;
 }
+#pragma clang diagnostic pop
```

Comment:
This method is currently not implemented or used, the code path that would call it, simply returns a method that says "--bind-local-port is not supported"